code/game/machinery/computer/security.dm

```diff
@@ -940,10 +940,10 @@ What a mess.*/
 									active1.fields["trim"] = active1.fields["rank"]
 								else
 									message_admins("Warning: possible href exploit by [key_name(usr)] - attempted to set change a crew member rank to an invalid path: [path]")
-									log_game("Warning: possible href exploit by [key_name(usr)] - attempted to set change a crew member rank to an invalid path: [path]")
+									usr.log_message("possibly trying to href exploit - attempted to set change a crew member rank to an invalid path: [path]", LOG_ADMIN)
 							else if(!isnull(text))
 								message_admins("Warning: possible href exploit by [key_name(usr)] - attempted to set change a crew member rank to an invalid value: [text]")
-								log_game("Warning: possible href exploit by [key_name(usr)] - attempted to set change a crew member rank to an invalid value: [text]")
+								usr.log_message("possibly trying to href exploit - attempted to set change a crew member rank to an invalid value: [text]", LOG_ADMIN)
```

Comment:
May want to keep the warning thing

Reply:
hows this 
![image](https://user-images.githubusercontent.com/85910816/182518060-69acae93-d1bc-42c8-a6b3-d5b3aa2fd81b.png)